tests/results.cpp

```diff
@@ -2765,3 +2765,159 @@ TEMPLATE_TEST_CASE("results: aggregate", ResultsFromTable, ResultsFromQuery, Res
         }
     }
 }
+
+
+
+TEST_CASE("BindingContext is notified about delivery of change notifications") {
```

Comment:
This test case should be in realm.cpp as that's where the the other binding context tests are.